tasks/config.yml

```
@@ -65,6 +65,28 @@
   with_items: "{{ tdagent_plugins | default([]) }}"
   notify:
     - restart td-agent
+  when: ansible_distribution != 'Debian'
+
+- name: Install plugins (Debian)
+  gem:
+    name: "{{ item.split(':')[0] }}"
```

we can do like above, but change this into
```suggestion
    name: ""{{ 'fluent-plugin-systemd:1.0.5' if item.split(':')[0] == 'fluent-plugin-systemd' else item }}""
```

because we only want to handle this, right?, this one much simpler